api/proto/ci/jenkins.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+package proto.ci;
+
+option go_package = "github.com/spinnaker/kleat/api/client/ci";
+
+import "permissions.proto";
+
+// Configuration for Jenkins.
+message Jenkins {
+  // Whether Jenkins is enabled.
+  bool enabled = 1;
+
+  // The list of configured Jenkins accounts.
+  repeated JenkinsAccount masters = 2;
+}
+
+// Configuration for a Jenkins account.
+message JenkinsAccount {
+  // The name of the account.
+  string name = 1;
+
+  // (Required) The username of the Jenkins user to authenticate as.
+  string username = 2;
+
+  // (Required) The password of the Jenkins user to authenticate as.
+  string password = 3;
+
+  // (Required) The address at which the Jenkins master is reachable.
```

Comment:
Should we call this a Jenkins server?  (And maybe in the config field too, though I guess that would break existing configs.)

Reply:
So the only reason I left this one as `master` but changed all the other references to that word is because the Jenkins docs are the only docs that actually use that word to refer to a Jenkins instance, but you're right and server is just as good so I changed it.

I didn't change the field name (here or on the other CI protos) because it seems like it's a slippery slope from there to renaming all the other poorly-named fields...

Reply:
Yeah, good point on renaming the fields 👍